spec/sidekiq-scheduler/job_presenter_spec.rb

```diff
@@ -1,116 +1,99 @@
-require "sidekiq-scheduler/job_presenter"
+require 'sidekiq-scheduler/job_presenter'
 
 describe SidekiqScheduler::JobPresenter do
-  let(:job_name) { "job_name" }
-  let(:job_config) { { 'cron' => '* * * * *', 'class' => 'SomeIvarJob', 'args' => '/tmp' } }
+  subject(:job_presenter) { described_class.new(job_name, attributes) }
+  let(:job_name) { 'job_name' }
   let(:attributes) { {} }
 
-  subject { described_class.new(job_name, attributes) }
   before { Sidekiq.redis(&:flushall) }
 
-  describe "#next_time" do
-    context "when the job has not a next time in redis" do
-      it "returns nothing" do
-        expect(subject.next_time).not_to be
-      end
+  describe '#next_time' do
+    subject { job_presenter.next_time }
+
+    context "when the job doesn't have a next time in redis" do
+      it { is_expected.to be_nil }
     end
 
-    context "when the job has next time in redis" do
+    context 'when the job has a next time in redis' do
       before { Sidekiq::Scheduler.update_job_next_time(job_name, next_time) }
       let(:next_time) { Time.now }
 
-      it "returns the parsed value" do
-        expect(subject.next_time).to eq(subject.relative_time(next_time))
-      end
+      it { is_expected.to eq(job_presenter.relative_time(next_time)) }
     end
   end
 
-  describe "#interval" do
-    context "when the attributes has a cron key" do
-      let(:attributes) { { "cron" => cron_value } }
-      let(:cron_value) { "cron_value" }
+  describe '#interval' do
```

Comment:
There's no base case for these set of tests; either interval, nor every are edge cases.

So, in this situation, it would be ok to have smth like:

``` ruby
context 'with "cron" key' do
  // ...
end

context 'with "interval" key' do
  // ...
end

context 'with "every" key' do
  // ...
end
```